changecalculator/src/main/java/org/msstate/cse/changecalculator/hibernate/model/SvnHistroy.java

```diff
@@ -0,0 +1,30 @@
+package org.msstate.cse.changecalculator. hibernate.model;
+

```

Comment:
The method and its class is spelled incorrectly consistently through the program.